src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -131,13 +131,13 @@ export class QuestionSubmissionFormComponent implements DoCheck {
 
     const indexes: Map<String, number> = new Map();
     this.model.recipientList.forEach((recipient: FeedbackResponseRecipient, index: number) => {
-      indexes.set(recipient.recipientIdentifier, index);
+      indexes.set(recipient.recipientIdentifier, index + 1);
     });
```

Comment:
> The index made to index+1 since it will conflict with the first element and result in lexicographically unsorted order.

Can you explain what do you mean by conflict with the first element? Also it would be good if you could attach a video or gif showing the result.

Reply:
Hi @jianhandev 
I have attached a video link showing the result of the fix.
[Demo for the fix](https://youtu.be/FIcgHVc03lU)

Reply:
@jianhandev The index is set to start from 1 instead of 0 for the line:
`indexes.set(recipient.recipientIdentifier, index + 1);`
Since if it starts from 0, the following line for the zero index defaults to Number.MAX_SAFE_INTEGER 
`indexes.get(firstRecipient.recipientIdentifier) || Number.MAX_SAFE_INTEGER;` 
and the first element after sorting will appear at the end.

Reply:
Thanks for the great explanation and video, I understand now. By the way, you could also drag and drop mp4 files to github comment boxes like this to insert a video directly.

Reply:
Hi @jianhandev 
Thanks for approving the request.
The video was too large to drag and keep in the comments. That's why I chose to upload it on youtube.